src/TextRenderer.vala

```diff
@@ -20,8 +20,8 @@ namespace Files {
     public class TextRenderer: Gtk.CellRendererText {
 
         const int MAX_LINES = 5;
-        private int border_radius;
-        private int double_border_radius;
+        public int border_radius { get; private set; }
```

Comment:
I don't see this property ever used outside of TextRenderer. This could probably stay private.

Reply:
Yeah, I ended up using only `double_border_radius`. I'll fix.